restio/model.py

```diff
@@ -91,7 +99,24 @@ def _update_meta(_meta: ModelMeta, extend: bool):
             if field_value.pk:
                 meta.primary_keys[field_name] = field_value
 
-        return super().__new__(cls, name, bases, dct)
+        # set alias name to class name when None
+        name_alias = meta.alias if meta.alias else name
```

Comment:
It's a bit nitpicky, but I think this can be shortened to `name_alias = meta.alias or name`